ckanext/showcase/i18n/es/LC_MESSAGES/ckanext-showcase.po

```diff
@@ -0,0 +1,521 @@
+# Translations template for ckanext-showcase.
+# Copyright (C) 2015 ORGANIZATION
+# This file is distributed under the same license as the ckanext-showcase
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2015.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: ckanext-showcase 1.0.0b1\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2015-09-23 14:49+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 0.9.6\n"
+
+#: ckanext/showcase/controller.py:56
+msgid "Unauthorized to create a package"
+msgstr "No tiene permisos para crear paqueges"
```

Comment:
typo: should say "paquetes", and actually we shouldn't display packages on the frontend but datasets ("conjuntos de datos")